nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -316,6 +322,223 @@ $(CsWinRTInternalProjection)
+      <_StubExeMsvcArgs Condition="'$(CsWinRTUseEnvironmentalTools)' != 'true'" Include='"$(_WindowsSdk-um-LibWildcardPath)"' />
+
+      <!-- Hide the copyright banner, to reduce visual clutter (https://learn.microsoft.com/cpp/build/reference/nologo-suppress-startup-banner-c-cpp) -->
+      <_StubExeMsvcArgs Include="/nologo" />
+
+      <!-- If not using the environmental tools, also pass the paths to the necessary include folders -->
+      <_StubExeMsvcArgs Condition="'$(CsWinRTUseEnvironmentalTools)' != 'true'" Include='/I "$(_MsvcIncludePath)"' />
+      <_StubExeMsvcArgs Condition="'$(CsWinRTUseEnvironmentalTools)' != 'true'" Include='/I "$(_WindowsSdk-ucrt-IncludePath)"' />
+      <_StubExeMsvcArgs Condition="'$(CsWinRTUseEnvironmentalTools)' != 'true'" Include='/I "$(_WindowsSdk-um-IncludePath)"' />
+
+      <!--
+        Configure the binary to statically link the C runtime library (https://learn.microsoft.com/cpp/build/reference/md-mt-ld-use-run-time-library).
+        This library (UCRT) has the implementation of all standard C runtime methods, like 'printf'. We want to actually link it dynamically, so we
+        will opt-in that individual library below. The dynamically linked version ships in the OS as 'ucrt.dll'. This matches the behavior of
+        Native AOT as well, which links UCRT dynamically, and VCRUNTIME140 (the MSVC-specific part of the C runtime) statically. The latter is much
+        smaller in size, so it does not matter. Dynamically linking UCRT reduces the size from 98 KB to 20 KB. The reason why we're not configuring
+        libraries to be dynamically linked, and then just having an opt-out for VCRUNTIME140, is that doing so causes a linker warning.
+      -->
+      <_StubExeMsvcArgs Condition="'$(Configuration)' == 'Debug'" Include="/MTd" />
+      <_StubExeMsvcArgs Condition="'$(Configuration)' != 'Debug'" Include="/MT" />
+
+      <!-- Optimize for speed (https://learn.microsoft.com/cpp/build/reference/o1-o2-minimize-size-maximize-speed) -->
+      <_StubExeMsvcArgs Include="/O2" />
+
+      <!-- Denote the start of the sequence of linker options (https://learn.microsoft.com/cpp/build/reference/compiler-command-line-syntax) -->
+      <_StubExeMsvcArgs Include="/link" />
+
+      <!-- Hide the copyright banner for the linker as well -->
+      <_StubExeMsvcArgs Include="/NOLOGO" />
+
+      <!-- Skip generating a manifest file, matching Native AOT (https://learn.microsoft.com/cpp/build/reference/manifest-create-side-by-side-assembly-manifest) -->
+      <_StubExeMsvcArgs Include="/MANIFEST:NO" />
+
+      <!--
+        Change the behavior for UCRT to be dynamically linked (the default would be to statically link it, due to '/MT[d]').
+        See: https://learn.microsoft.com/cpp/build/reference/defaultlib-specify-default-library.
+        Also see: https://learn.microsoft.com/cpp/build/reference/nodefaultlib-ignore-libraries.
+      -->
+      <_StubExeMsvcArgs Include="/NODEFAULTLIB:libucrt.lib" />
```

Comment:
Do we need to consider debug versions here or no?

Reply:
Not sure. The stub is always built with `\O2` anyway 🤔

Reply:
Getting this when publishing in Debug, I assume this is what you meant?

> "LINK : warning LNK4098: defaultlib 'ucrt.lib' conflicts with use of other libs; use /NODEFAULTLIB:library"